rpc/transaction_test.go

```diff
@@ -118,28 +95,70 @@ func TestTransactionByBlockIdAndIndex(t *testing.T) {
 		ExpectedTxn Transaction
 	}
 
-	var InvokeTxnV1example = InvokeTxnV1{
+	var InvokeTxnV3example = InvokeTxnV3{
 		Type:    TransactionType_Invoke,
-		MaxFee:  utils.TestHexToFelt(t, "0x53685de02fa5"),
-		Version: TransactionV1,
-		Nonce:   &felt.Zero,
+		Version: TransactionV3,
+		Nonce:   utils.TestHexToFelt(t, "0x359d"),
 		Signature: []*felt.Felt{
-			utils.TestHexToFelt(t, "0x4a7849de7b91e52cd0cdaf4f40aa67f54a58e25a15c60e034d2be819c1ecda4"),
-			utils.TestHexToFelt(t, "0x227fcad2a0007348e64384649365e06d41287b1887999b406389ee73c1d8c4c"),
+			utils.TestHexToFelt(t, "0x665f0c67ed4d9565f63857b1a55974b98b2411f579c53c9f903fd21a3edb3d1"),
+			utils.TestHexToFelt(t, "0x549c4480aba4753c58f757c92b5a1d3d67b2ced4bf06076825af3f52f738d6d"),
+		},
+		SenderAddress:         utils.TestHexToFelt(t, "0x143fe26927dd6a302522ea1cd6a821ab06b3753194acee38d88a85c93b3cbc6"),
+		NonceDataMode:         DAModeL1,
+		FeeMode:               DAModeL1,
+		PayMasterData:         []*felt.Felt{},
+		AccountDeploymentData: []*felt.Felt{},
+		ResourceBounds: ResourceBoundsMapping{
+			L1Gas: ResourceBounds{
+				MaxAmount:       "0x3bb2",
+				MaxPricePerUnit: "0x2ba7def30000",
+			},
+			L2Gas: ResourceBounds{
+				MaxAmount:       "0x0",
+				MaxPricePerUnit: "0x0",
+			},
 		},
-		SenderAddress: utils.TestHexToFelt(t, "0x315e364b162653e5c7b23efd34f8da27ba9c069b68e3042b7d76ce1df890313"),
 		Calldata: []*felt.Felt{
 			utils.TestHexToFelt(t, "0x1"),
-			utils.TestHexToFelt(t, "0x13befe6eda920ce4af05a50a67bd808d67eee6ba47bb0892bef2d630eaf1bba"),
+			utils.TestHexToFelt(t, "0x6b74c515944ef1ef630ee1cf08a22e110c39e217fa15554a089182a11f78ed"),
+			utils.TestHexToFelt(t, "0xc844fd57777b0cd7e75c8ea68deec0adf964a6308da7a58de32364b7131cc8"),
+			utils.TestHexToFelt(t, "0x13"),
+			utils.TestHexToFelt(t, "0x41bbf1eff2ac123d9e01004a385329369cbc1c309838562f030b3faa2caa4"),
+			utils.TestHexToFelt(t, "0x54103"),
+			utils.TestHexToFelt(t, "0x7e430a7a59836b5969859b25379c640a8ccb66fb142606d7acb1a5563c2ad9"),
+			utils.TestHexToFelt(t, "0x6600d829"),
+			utils.TestHexToFelt(t, "0x103020400000000000000000000000000000000000000000000000000000000"),
```

Comment:
Shouldn't this call data be like this https://sepolia.voyager.online/tx/0x6a4a9c4f1a530f7d6dd7bba9b71f090a70d1e3bbde80998fde11a08aab8b282